.github/workflows/npmpublish.yml

```diff
@@ -49,3 +56,43 @@ jobs:
       - run: ./scripts/deploy-docs.sh "${GITHUB_REF:1}"
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+      - name: Upload chart.js
+        id: upload-chart-js
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_AUTH_TOKEN }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ./dist/chart.js
+          asset_name: chart.js
+          asset_content_type: text/javascript
+      - name: Upload chart.min.js
+        id: upload-chart-min-js
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_AUTH_TOKEN }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ./dist/chart.min.js
```

Comment:
glob not supported?

Reply:
Nope 😦. https://github.com/actions/upload-release-asset/issues/9